.gitignore

```diff
@@ -21,6 +21,3 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-
-.eslintrc.json
-.stylelintrc.json
```

Comment:
I think we should not push the local files to keep our code clean. (eslintrc.json, stylelintrc, and .babelrc files in this case).